src/integrationTest/groovy/wooga/gradle/unity/UnityTaskIntegrationSpec.groovy

```diff
@@ -76,9 +76,12 @@ abstract class UnityTaskIntegrationSpec<T extends UnityTask> extends UnityIntegr
         where:
         [testCase, useSetter, value] <<
                 [
+                        // For each option that is a flag (with a boolean value)
                         UnityCommandLineOption.flags.collect(
                                 { it -> ["${it}", it.flag] }),
+                        // Test with and without the setter
                         [true, false],
+                        // Test both true and false values
```

Comment:
Well I think that commends won't make this more readable. Again that is why I prefer the table definitions since it might be more verbose but it is clearer which variable will have which value (in most cases)
The commends here now won't help really to see what you want to achieve with the nested array